src/js/models/documentexports.js

```diff
@@ -48,6 +49,35 @@ define(function (require, exports, module) {
         layerExportsMap: Immutable.Map()
     });
 
+    Object.defineProperties(DocumentExports.prototype, objUtil.cachedGetSpecs({
+        /**
+         * Produce an array of ExportAssets for the root of this document, converted to a JS object
+         * This is used for serializing the data to Ps metadata store
+         *
+         * @return {Array.<object>} An array of ExportAsset-like JS objects
+         */
+        rootExportsArray: function () {
+            if (this.rootExports && this.rootExports.size > 0) {
+                return this.rootExports.toJS();
```

Comment:
I removed `rootExportsArray` entirely as it is both unused, and silly in hindsight.